cluster/saltbase/salt/top.sls

```diff
@@ -13,6 +13,7 @@ base:
   'roles:kubernetes-pool':
     - match: grain
     - docker
+    - flannel
```

Comment:
should this be guarded by {% if pillar.get('network_provider', '').lower() == 'flannel' %}